src/utils/SpoonacularRequests.ts

```diff
@@ -1,8 +1,5 @@
-import { useContext } from 'react';
-import { FoodprintContext } from '../store/ingredient-context';
 
-export const fetchData = async() => {
-  const ingredients = useContext(FoodprintContext).ingredients.items;
+export const fetchData = async(addRecipe: (t:string, i:number, im: string)=> void, ingredientList: any) => {
```

Comment:
could you make these parameters more readable for other devs

not just t, or i